src/Okta.Sdk/Generated/IlogClientInfo.Generated.cs

```diff
@@ -5,10 +5,6 @@
 
 // This file was automatically generated. Don't modify it directly.
 
-using System;
```

Comment:
These are included in the output of `npm run generate` and should not be removed.